nav2_lifecycle_manager/include/nav2_lifecycle_manager/lifecycle_manager.hpp

```diff
@@ -31,11 +31,22 @@ namespace nav2_lifecycle_manager
 {
 
 using nav2_msgs::srv::ManageLifecycleNodes;
-
+/**
+ * @class nav2_lifecycle_manager::LifecycleManager
+ * @brief Implements service interface to control the lifecycle nodes of
+ * Navigation2 stack. It receives control request and then uses lifecycle
```

Comment:
I think the word **transition** is more appropriate in this case as opposed to control (e.g. "...to **transition** the lifecycle nodes" and "It receives **transition** request...").